webapp/src/app.tsx

```diff
@@ -862,6 +864,8 @@ export class ProjectView
         if (options.filesOverride)
             Util.jsonCopyFrom(files, options.filesOverride)
         files["pxt.json"] = JSON.stringify(cfg, null, 4) + "\n"
+        //TODO this
+        files["serial.json"] = JSON.stringify({"pineapple": "banana"}) + "\n"
```

Comment:
serial.txt?

Reply:
sure - the file is for all practical purposes empty, so can have whatever extension we want.